src/features/courses/course-controller.ts

```diff
@@ -1269,8 +1297,10 @@ class CourseController {
         }
 
         let showCorrectAnswers = false;
+        let answersSubmitted: number | undefined;
         if (options.role === Role.PROFESSOR && !_.isNil(workbook)) {
             showCorrectAnswers = true;
+            answersSubmitted = 1;
```

Comment:
Is this a new parameter, even though Perl doesn't type booleans (which makes me 👿 ) I have been typing them as booleans and converting them at the renderer level

When I first looked at this I thought that field represented a count and couldn't understand why you were resetting it)